go/logic/ccql.go

```diff
@@ -2,28 +2,32 @@ package logic
 
 import (
 	"fmt"
+	"github.com/outbrain/golib/sqlutils"
 	"log"
 	"strings"
-	"github.com/outbrain/golib/sqlutils"
 	"sync"
 )
 
 // queryHost connects to a given host, issues the given set of queries, and outputs the results
 // line per row in tab delimited format
-func queryHost(host string, user string, password string, schema string, queries []string, timeout float64) error {
+func queryHost(host string, user string, password string, schema string, queries []string, timeout float64, viewSourceSchema bool) error {
 	mysqlURI := fmt.Sprintf("%s:%s@tcp(%s)/%s?timeout=%fs", user, password, host, schema, timeout)
 	db, _, err := sqlutils.GetDB(mysqlURI)
 	if err != nil {
 		return err
 	}
-
+	fmt.Println()
```

Comment:
Why is this `Println()` added?

Reply:
When using the flag --ask-pass the results are shown are next to the Enter Password phrase. Will try to make this formatting better.